js/layout-javascript/agenda-code.js

```diff
@@ -509,6 +518,16 @@ DynamicList.prototype.scrollEvent = function() {
   });
 }
 
+DynamicList.prototype.updateNavigatorQuery = function(defaultDateIndex, indexOfClickedDate) {
+  if (defaultDateIndex === indexOfClickedDate) {
+    Fliplet.Page.Context.remove('dayUserLeft');
+  } else {
+    Fliplet.Page.Context.update({
+      dayUserLeft: indexOfClickedDate
```

Comment:
Please use the `dateIndex` key. This means we also add the capability of taking users to a date via index.

```suggestion
      dateIndex: indexOfClickedDate
```